From bd4f218402afaf2d76caf651eac13c743b735604 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Tue, 8 Sep 2020 16:22:26 +0800
Subject: [PATCH] Test case for Delete included .asm changing - asm file

Description: Delete included ForAddTest.asm file in the TestPkg/FakeEmuSec/Ia32/SwitchRam.asm.

Repo: edk2

Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 TestPkg/FakeEmuSec/Ia32/SwitchRam.asm | 1 -
 1 file changed, 1 deletion(-)

diff --git a/TestPkg/FakeEmuSec/Ia32/SwitchRam.asm b/TestPkg/FakeEmuSec/Ia32/SwitchRam.asm
index a829fbd3aa..9a3139de6b 100644
--- a/TestPkg/FakeEmuSec/Ia32/SwitchRam.asm
+++ b/TestPkg/FakeEmuSec/Ia32/SwitchRam.asm
@@ -65,7 +65,6 @@ SecSwitchStack   PROC
     ;
     ; Fixup the ebp point to permanent memory
     ;
-    #include "ForAddTest.asm"
 
     ;
     ; Fixup callee's ebp point for PeiDispatch
-- 
2.27.0.windows.1

